### PR TITLE
Added toast scenarios

### PR DESCRIPTION
**Pull request by phoenixblast**

### CODE REVIEW

File: src/wintoastlib.h [hunk 1]
```diff
@@ -142,6 +145,7 @@ namespace WinToastLib {
         std::wstring                        _imagePath{};
         std::wstring                        _audioPath{};
         std::wstring                        _attributionText{};
+        std::wstring                        _scenario = L"Default";
```
Comment (mohabouje) [inline src/wintoastlib.h:148]: Following the other members, lets use aggregate initialization

File: src/wintoastlib.cpp [hunk 2]
```diff
@@ -1112,6 +1148,11 @@ const std::wstring& WinToastTemplate::attributionText() const {
     return _attributionText;
 }
 
+const std::wstring& WinToastLib::WinToastTemplate::scenario() const
```
Comment (mohabouje) [inline src/wintoastlib.cpp:1151]:
```
const std::wstring& WinToastLib::WinToastTemplate::scenario() const
{
```
to
```
const std::wstring& WinToastLib::WinToastTemplate::scenario() const {
```

File: src/wintoastlib.cpp [hunk 3]
```diff
@@ -1065,6 +1091,16 @@ void WinToastTemplate::setExpiration(_In_ INT64 millisecondsFromNow) {
     _expiration = millisecondsFromNow;
 }
 
+void WinToastLib::WinToastTemplate::setScenario(Scenario scenario)
```
Comment (mohabouje) [inline src/wintoastlib.cpp:1094]:
```
void WinToastLib::WinToastTemplate::setScenario(Scenario scenario)
{
```
to 
```
void WinToastLib::WinToastTemplate::setScenario(Scenario scenario) {
```

File: src/wintoastlib.h (hunk 1 above)
Comment (phoenixblast) [reply src/wintoastlib.h]: Done

File: src/wintoastlib.cpp (hunk 2 above)
Comment (phoenixblast) [reply src/wintoastlib.cpp]: Done

File: src/wintoastlib.cpp (hunk 3 above)
Comment (phoenixblast) [reply src/wintoastlib.cpp]: Done

**Pull request merged**